src/components/Drawer/__tests__/Drawer.spec.tsx

```diff
@@ -1,108 +1,104 @@
 import React from 'react';
-import toJson from 'enzyme-to-json';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
 import { Drawer } from '../Drawer';
 
 jest.mock('react-transition-group', () => {
     const FakeTransition = jest.fn(({ children }) => children({ state: 'entered' }));
     return { Transition: FakeTransition };
 });
+const onClickMock = jest.fn();
 
 describe('Drawer', () => {
     it('should render correctly', () => {
-        const wrapper = mount(<Drawer title="some title">some text</Drawer>);
-        expect(toJson(wrapper)).toMatchSnapshot();
+        const view = render(<Drawer title="some title">some text</Drawer>);
+
+        expect(view.container).toMatchSnapshot();
+        expect(screen.getByRole('heading', { name: 'some title' })).toBeInTheDocument();
     });
 
     it('should pass initial expand status', () => {
-        const wrapper = mount(
-            <Drawer initialIsExpanded title="some title">
+        render(
+            <Drawer onClick={onClickMock} initialIsExpanded title="some title">
                 some text
             </Drawer>
         );
+        const expandButton = screen.getByRole('button', { name: '' });
 
-        expect(wrapper.find('.Drawer').hasClass('Drawer--isShownAndExpanded')).toBe(true);
-        expect(wrapper.find('.Drawer').hasClass('Drawer--isShownAndClosed')).toBe(false);
-        expect(
-            wrapper.find('.Drawer__expandButton').hasClass('Drawer__expandButton--isExpanded')
-        ).toBe(true);
+        expect(expandButton).toBeInTheDocument();
+        expect(onClickMock).not.toHaveBeenCalled();
```

Comment:
why do we need this check? If we didn't do any user interaction, would the code trigger itself?

Reply:
:+1: